frontend/src/static/js/components/test/webstatus-stats-missing-one-impl-chart-panel.test.ts

```diff
@@ -179,4 +200,94 @@ describe('WebstatusStatsMissingOneImplChartPanel', () => {
       'first row ID',
     );
   });
+
+  it('assert correct getMissingOneImplementationFeatures calls', async () => {
+    apiClientStub.getMissingOneImplementationFeatures.resolves([
+      {
+        feature_id: 'css',
+      },
+      {
+        feature_id: 'html',
+      },
+      {
+        feature_id: 'js',
+      },
+      {
+        feature_id: 'bluetooth',
+      },
+    ]);
+    const chart = el.shadowRoot!.querySelector(
+      '#missing-one-implementation-chart',
+    )!;
+
+    const chartClickEvent: ChartSelectPointEvent = new CustomEvent(
+      'point-selected',
+      {
+        detail: {
+          label: 'Safari',
+          timestamp: new Date('2024-01-01'),
+          value: 123,
+        },
+        bubbles: true,
+      },
+    );
+    // Simulate point-selected event on the chart component
+    chart.dispatchEvent(chartClickEvent);
+    await el.updateComplete;
+
+    expect(el._pointSelectedTask).to.exist;
+    expect(
+      apiClientStub.getMissingOneImplementationFeatures,
+    ).to.have.been.calledWith(
+      'safari',
+      ['chrome', 'firefox'],
+      new Date('2024-01-01'),
+    );
+
+    const chartClickEventOne: ChartSelectPointEvent = new CustomEvent(
+      'point-selected',
+      {
+        detail: {
+          label: 'Firefox',
+          timestamp: new Date('2024-01-01'),
+          value: 123,
+        },
+        bubbles: true,
+      },
+    );
+    chart.dispatchEvent(chartClickEventOne);
+    await el.updateComplete;
+
+    expect(el._pointSelectedTask).to.exist;
+    expect(
+      apiClientStub.getMissingOneImplementationFeatures,
+    ).to.have.been.calledWith(
+      'firefox',
+      ['chrome', 'safari'],
+      new Date('2024-01-01'),
+    );
+
+    const chartClickEventTwo: ChartSelectPointEvent = new CustomEvent(
+      'point-selected',
+      {
+        detail: {
+          label: 'Chrome',
```

Comment:
Feel free to fix this in another PR but it should be Chromium
```suggestion
          label: 'Chromium',
```